app/src/main/java/org/covidwatch/android/data/backend/BackendConstants.kt

```diff
@@ -1,9 +1,9 @@
-package org.covidwatch.android.data.firestore
+package org.covidwatch.android.data.backend
 
 import java.util.*
 import java.util.concurrent.TimeUnit
 
-object FirestoreConstants {
+object BackendConstants {
```

Comment:
Similar to previous point... instead of just "Backend" can we specify what this is doing?

Reply:
Renamed it back to FirestoreConstants and moved the two items that are not firebase related to the location where they are used.